tests/core/ssl/test_ssl.py

```diff
@@ -9,10 +9,15 @@
 from chia.server.ssl_context import chia_ssl_ca_paths, private_ssl_ca_paths
 from chia.server.ws_connection import WSChiaConnection
 from chia.ssl.create_ssl import generate_ca_signed_cert
+from chia.types.blockchain_format.sized_bytes import bytes32
 from chia.types.peer_info import PeerInfo
 from chia.util.ints import uint16
 
 
+def dummy_close_callback(_: WSChiaConnection, __: int) -> None:
+    pass
+
+
```

Comment:
Changed it to be optional in 30a6679a1f5b7b7a7d2d1dda969d57431e310783.

Reply:
So i guess the cost here is that we risk forgetting to consider having a close callback in some future use.  This could be avoided by not providing a default.  Does that sound safer, while retaining the ease of skipping it?

Reply:
See 01df65fe4eed2e4e6f6cd7042d934a182743b5d9